.azuredevops/platformPipelines/platform.dependencies.yml

```
@@ -24,7 +24,7 @@ trigger: none
 #       - 'utilities/pipelines/dependencies/**'
 
 variables:
-  - template: '/.azuredevops/pipelineVariables/global.variables.yml'
+  - template: '../../global.variables.yml'
```

Shouldn't this work too for all references?

```suggestion
  - template: './global.variables.yml'
```

---

I had that before and the pipelines fail to start because the starting path is where the pipeline is stored. Error:


/.azuredevops/modulePipelines/ms.authorization.policyassignments.yml: Could not find /.azuredevops/modulePipelines/global.variables.yml in repository self hosted on https://github.com/ using commit a84a6d28484dc3c02f100c737950aedbf6a2340d. GitHub reported the error, "Not Found"

---

Alright then. A shame GitHub & Azure DevOps don't work the same